### PR TITLE
UI network updates

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Updates to wording on where users need to look to find information about billable usage.

# Reasons
The UI changed.

### CODE REVIEW

**Comment by devinmarieb** [comment]
@stalmy Plz ignore the other PR as I've now added that change into this larger PR around the Storage/Network UI update. The images have been removed and new ones will be added soon in a separate request.

File: jekyll/_cci2/persist-data.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The information below describes how your network and storage usage is accumulati
 
 **NOTE:** Your overall **Network Transfer** amount is not representative of your billable usage. Only certain actions will result in network egress, which in turn results in billable usage. Details of these actions are described below.
 
-### Overview of storage and network transfer
+### Overview of all storage and network transfer
```
Comment (thomasr8) [inline jekyll/_cci2/persist-data.md:66]:
```suggestion
### Overview of all storage and network transfers
```

Comment (thomasr8) [reply jekyll/_cci2/persist-data.md:66]: Should this be "transfers"?

**Review by thomasr8 — APPROVED** [review]
Looks good! Just one minor comment.

File: jekyll/_cci2/persist-data.md (hunk 1 above)
Comment (devinmarieb) [reply jekyll/_cci2/persist-data.md:66]: @stalmy what do you think? I also added the "all" to specify that we are not just talking about billable usage, but of both overall N&S as well as billable N&S.

**Pull request merged**